### DISCUSSION TITLE
Superfences don't work any more on GitHub Pages ?

### DISCUSSION THREAD

**Discussion by glassus** [facelessuser/pymdown-extensions#1519] [Q&A] [answered]
Hi,
Since this morning, the superfences on my GithubPages don't appear anymore.

On my local website :
![image](https://user-images.githubusercontent.com/54031955/143878332-f5e22753-8c46-42b9-88b6-200c16897ec4.png)

On my GithubPages (https://glassus.github.io/premiere_nsi/)
![image](https://user-images.githubusercontent.com/54031955/143878451-a7d41a9f-28af-4ac2-8d3f-9dcced0e0c3d.png)

**Comment by facelessuser** [comment, marked as answer]
I'm not seeing a difference in anything dealing with SuperFences. It looks like, maybe, you are seeing a difference in the "Tabbed" extension? You also seem to be using MkDocs Material. The reality is that this is an issue with MkDocs Material. I should rephrase, not a problem, but an incompatibility. MkDocs Material changed to use the new experimental "alternative tab styling" in version 8.0. A migration guide is provided here: https://squidfunk.github.io/mkdocs-material/upgrade/#changes-to-mkdocsyml.

**Comment by glassus** [comment]
Thank you so much !
Sorry for the confusion between Tabbed and Superfences.
And you were right, just adding :
```
markdown_extensions:
    - pymdownx.tabbed:
          alternate_style: true 
```
(as said in your link)
solved the problem !

Thanks again,

Gilles
